app/services/arcgis_api/suggestion.rb

```diff
@@ -0,0 +1,3 @@
+module ArcgisApi
+  Suggestion = Struct.new(:text, :magic_key, keyword_init: true)
```

Comment:
I would consider just defining this inside the client class

Reply:
Maybe just for convenience, it's tied very closely to the API output of that one endpoint, I don't expect we'll be constructing or using these outside of that class (or a mocked version of that class)